src/store/store.js

```diff
@@ -48,6 +49,10 @@ export default new Store({
 
 		isPublic: (_state, get) =>
 			!!get.shareTokenParam,
+
+		printLink: (_state, get) => get.isPublic
+			? generateUrl(`/apps/collectives/p/${get.shareTokenParam}/print/${get.currentCollective.name}`)
```

Comment:
`get.currentCollective` might be unset when you don't have a collective open. Also, this would always print the current collective. So better to pass the collective as parameter here and use it.